example/main.go

```diff
@@ -19,9 +19,9 @@ import (
 	"log"
 	"time"
 
+	"contrib.go.opencensus.io/exporter/zipkin"
 	openzipkin "github.com/openzipkin/zipkin-go"
 	"github.com/openzipkin/zipkin-go/reporter/http"
-	"go.opencensus.io/exporter/zipkin"
```

Comment:
Will this url continue to work?

Reply:
I just replaced it in 4th commit https://github.com/census-ecosystem/opencensus-go-exporter-zipkin/pull/3/commits/9b74147beedeedd7c60a42a33f9566e9410ba524